### PR TITLE
visualize keylifecycle event.Message

### PR DESCRIPTION
**Pull request by beilineili**
Change podphase.go, add event message into tags, to visual keylifecycle event.Message.

### CODE REVIEW

**Review by d3c3mber — APPROVED** [review]
LGTM

File: internal/grafanadi/service/podphase.go [hunk 1]
```diff
@@ -85,6 +85,8 @@ func ConvertPodPhase2Frame(podPhases []*storagemodel.LifePhase) model.DataFrame
 				if msg != "" && msg != nil {
 					phaseMessage = msg.(string)
 				}
+				// 如果是eventObject则赋值message
```
Comment (larryck) [inline internal/grafanadi/service/podphase.go:88]: Please do not use Chinese characters in source files

Comment (beilineili) [reply internal/grafanadi/service/podphase.go]: ok, i have deleted the Chinese characters

File: internal/grafanadi/service/podphase.go [hunk 2]
```diff
@@ -85,6 +85,7 @@ func ConvertPodPhase2Frame(podPhases []*storagemodel.LifePhase) model.DataFrame
 				if msg != "" && msg != nil {
 					phaseMessage = msg.(string)
 				}
+				op = msg.(string)
```
Comment (larryck) [inline internal/grafanadi/service/podphase.go:88]: LGTM

**Review by d3c3mber — APPROVED** [review]

**Review by linuzb — APPROVED** [review]
LGTM

**Pull request merged**